### DISCUSSION TITLE
Welcome to product_video_ads Discussions!

### DISCUSSION THREAD

**Discussion by roberto-goncalves** [google/product_video_ads#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexvandepol** [comment]
Hello all,

I'm a freelance PPC expert based in the Netherlands, focusing mainly on large-scale retail and travel advertising. My clients often require scalable video creation solutions that effectively highlight their key promotions during specific periods. Unfortunately, existing tools tend to be either too labor-intensive or fall short in meeting the necessary quality and brand standards. That's why I'm particularly keen on this feature, as it promises to address these challenges. While my expertise lies in marketing, I possess limited coding skills. However, I've collaborated extensively with developers on various Google APIs over the years, which may give me a unique perspective on this matter.
